optuna_dashboard/ts/components/PreferentialTrials.tsx

```diff
@@ -182,11 +182,15 @@ export const PreferentialTrials: FC<{ studyDetail: StudyDetail | null }> = ({
     return null
   }
   const theme = useTheme()
+  
+  const running_trials = studyDetail.trials.filter((t) => t.state === "Running")
+  const active_trials = running_trials.concat(studyDetail.best_trials)
```

Comment:
Could you use camelCase for variables?

```suggestion
  const runningTrials = studyDetail.trials.filter((t) => t.state === "Running")
  const activeTrials = running_trials.concat(studyDetail.best_trials)
```